tests/functional/test_freeze.py

```diff
@@ -782,3 +782,11 @@ def test_freeze_path_multiple(tmpdir, script, data):
         simple2==3.0
         <BLANKLINE>""")
     _check_output(result.stdout, expected)
+
+
+def test_freeze_direct_url_archive(script, data, with_wheel):
```

Comment:
Use the `shared_data` fixture to avoid the copy operation.

Reply:
done